src/mmw/apps/modeling/tasks.py

```diff
@@ -146,8 +146,19 @@ def analyze_pointsource(area_of_interest):
     return [point_source_pollution(area_of_interest)]
 
 
+def ensure_aoi_4d(area_of_interest):
```

Comment:
I think given that it's not checking for an anomalous data structure which _needs_ to be 4d, it makes sense for the `aoi_resolution` function to be able to handle both Single and Multi Polygons directly.  Or, if a forcing function is still preferred maybe a rename to `get_first_polygon_in_multipolygon` or something would clarify its purpose.